node/libs/crypto/src/bls12_381/mod.rs

```diff
@@ -153,22 +167,31 @@ impl Ord for Signature {
     }
 }
 
-/// Type safety wrapper around a `blst` signature indicating that it is an aggregated signature
-///
-/// Due to the `blst` aggregated signatures not having a verify method, this is stored converted to
-/// a bare signature internally.
-#[derive(Clone, Debug, PartialEq, Eq)]
-pub struct AggregateSignature(bls::Signature);
+/// Type safety wrapper around a `blst` aggregate signature
+#[derive(Clone, Debug)]
+pub struct AggregateSignature(bls::AggregateSignature);
 
-impl AggregateSignature {
-    /// Generates an aggregate signature from a list of signatures
-    pub fn aggregate<'a>(sigs: impl IntoIterator<Item = &'a Signature>) -> Result<Self, Error> {
-        let sigs: Vec<&bls::Signature> = sigs.into_iter().map(|s| &s.0).collect();
+impl Default for AggregateSignature {
+    fn default() -> Self {
+        // This can't fail in production since we are decoding a known value.
```

Comment:
I personally like the fact that we can have an empty signature, but isn't it against some RFC or sth?

Reply:
I don't think so. Technically the identity point is a valid signature, but I'll double check it.

Reply:
What does "empty signature" mean from cryptography perspective?
It cannot be empty. Since aggregation operation must have at least one signature as input.

Reply:
It's the identity point. Which in here corresponds to the Default aggregate signature.